### PR TITLE
Feat/gtw 8 add links to sign up organizations

### PR DESCRIPTION
**Pull request by boscocg**
# gtw 8 add links to sign up organizations

## Type of PR

- [x] - feature

### CODE REVIEW

File: src/components/templates/org/signup/signup.tsx [hunk 1]
```diff
@@ -218,10 +218,9 @@ export function OrgSignUpTemplate({
                     onClick={handlePrevious}
                     sx={{ height: 48 }}
                   >
-                    Back
+                    {t('action-back')}
```
Comment (abisarvepalli) [inline src/components/templates/org/signup/signup.tsx:221]: veryyy nice

**Review by abisarvepalli — COMMENTED** [review]
LGTM, 1 comment

File: src/components/templates/landing/investors/investors.tsx [hunk 2]
```diff
@@ -84,10 +85,9 @@ export const Investors = forwardRef<
                   },
                 })}
               >
-                <Image
+                <img
```
[the remark this answers is not in the document]
Comment (boscocg) [reply src/components/templates/landing/investors/investors.tsx:88]: I noticed that both in the dev environment and in production, the images of investors are not appearing on the landing page. I talked to Rafa (he who had made this component) and we identified that it was this tag.

[the remark this answers is not in the document]
Comment (abisarvepalli) [reply src/components/templates/landing/investors/investors.tsx:88]: Is `Image` used elsewhere? Does those locations require updates as well?

[the remark this answers is not in the document]
Comment (abisarvepalli) [reply src/components/templates/landing/investors/investors.tsx:88]: Approving this PR to unblock merge, but curious if this creates inconsistencies -- and how we should resolve them -- in the repo

[the remark this answers is not in the document]
Comment (boscocg) [reply src/components/templates/landing/investors/investors.tsx:88]: Strange, we need to investigate, but you can see here (https://www.mygateway.xyz/ - in the section Guardians of the Gateway). Maybe the problem is the width and height dynamics.

**Review by abisarvepalli — APPROVED** [review]

**Pull request merged**